src/main/java/seedu/address/logic/commands/listcommand/ListCommand.java

```diff
@@ -2,48 +2,66 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.Map;
+
+import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.entity.Entity;
+import seedu.address.model.entity.Participant;
+import seedu.address.model.entity.Team;
 
 /**
  * Lists every {@link Entity} (i.e. mentor, participant, team) in Alfred.
  */
 public class ListCommand extends Command {
 
-    /* Possible Fields */
+    public static final String MESSAGE_SUCCESS = "Listed all teams";
 
     public static final String COMMAND_TYPE = "list";
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        // Possible format
-        // TeamList teamList = model.getTeamList();
-
-        // I think Team should have a method that lists its connections (e.g. Team#showConnection)
-        // i.e.
         // Team: <Team Name> (Mentor: <Mentor Name>)
         // Participants:
         //    <Participant 1> (in this team)
         //    ...
         //    <Participant n>
+        model.getTeamList().list()
```

Comment:
Hmmm do you want to define this inside the Team Entity instead? Seems a bit weird to do it here

Reply:
Yea I think Team should have a showConnection method. I think it was in my previous comments too